app/javascript/components/NewsLetter.jsx

```diff
@@ -80,60 +74,36 @@ export default function NewsLetter() {
           adobeAnalyticsForNewsLetter();
         }
       }
-      return (
-        <VStack justifyContent="center" tabIndex="-1" id="news-letter-msg-box">
-          <Heading
-            level="h2"
-            size="heading6"
-            maxWidth="640px"
-            noSpace
-            textAlign="center"
-          >
-            Thank you for signing up to the MyLibraryNYC Newsletter!
-          </Heading>
-          <Text noSpace>
-            Check your email to learn about teacher sets, best practices &
-            exclusive events.
-          </Text>
-        </VStack>
-      );
-    } else {
-      return (
-        <VStack gap="m" justifyContent="center">
-          <Heading
-            level="h2"
-            size="heading6"
-            maxWidth="640px"
-            noSpace
-            textAlign="center"
-          >
-            Learn about new teacher sets, best practices &amp; exclusive events
-            when you sign up for the MyLibraryNYC Newsletter!
-          </Heading>
-          <Stack
-            direction={isLargerThanMobile ? "row" : "column"}
-            width="100%"
-            alignItems="flex-start"
-            justifyContent="center"
-          >
-            <TextInput
-              id="news-letter-text-input"
-              type="email"
-              labelText="Your email address"
-              showLabel={false}
-              maxWidth="500px"
-              placeholder="your email address"
-              width="100%"
-              onChange={handleNewsLetterEmail}
-              required
-              invalidText={message}
-              isInvalid={isInvalid}
-            />
-            {submitButtonaAndProgressBar()}
-          </Stack>
-        </VStack>
-      );
     }
+    return (
+      <NewsletterSignup
+        id="news-letter-text-input"
+        view={view}
+        isInvalidEmail={isInvalidEmail}
+        valueEmail={email}
+        onChange={handleNewsLetterEmail}
+        onSubmit={handleSubmit}
+        showPrivacyLink={false}
+        title="Sign Up for Our Newsletter"
+        descriptionText="Learn about new teacher sets, best practices & exclusive events when you sign up for the MyLibraryNYC Newsletter!"
```

Comment:
Add a comma after `best practices` and change the ampersand (&) to `and`.  Also, use a lowercase `n` for `newsletter`.